pkg/pipelines/init.go

```diff
@@ -20,11 +20,14 @@ import (
 
```

Comment:
I don't think you are changing the "init" command, right?

Reply:
No we are not, I guess it's a bad merge.. will check it

Reply:
pipelines and templates need to be added